### ISSUE TITLE
Check hadd events vs meta nevents

### ISSUE THREAD

**Issue by hschellman** [DUNE/NTupleProd#8]
End of hadd jobs needs to check that events are the same in file and metadata

**Comment by calcuttj** [comment]
We noticed hadded file and metadata had different number of events. I found the reason. 

Part of the [PDSPAnalyzer module](https://cdcvs.fnal.gov/redmine/projects/protoduneana/repository/entry/protoduneana/singlephase/Analysis/PDSPAnalyzer_module.cc?utf8=%E2%9C%93&rev=v09_32_00)  checks if there was a beam line particle that created the event, if not it skips the event and doesn't fill the output tree. The TFileMetadataDUNE service counts the number of events that pass the filters before the module is run so this count can be different than the entries in the tree.

To-do: in lar_wrapper.py, get the number of events from the output ntuple and store in the metadata
